clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/DefaultMigrateSecurityGroupStrategy.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.aws.deploy.handlers;
+
+import com.amazonaws.services.ec2.model.SecurityGroup;
+import com.netflix.spinnaker.clouddriver.aws.deploy.ops.securitygroup.MigrateSecurityGroupReference;
+import com.netflix.spinnaker.clouddriver.aws.deploy.ops.securitygroup.SecurityGroupLookupFactory.SecurityGroupUpdater;
+import com.netflix.spinnaker.clouddriver.aws.deploy.ops.securitygroup.SecurityGroupLookupFactory.SecurityGroupLookup;
+import com.netflix.spinnaker.clouddriver.aws.deploy.ops.securitygroup.SecurityGroupMigrator.SecurityGroupLocation;
+import com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class DefaultMigrateSecurityGroupStrategy implements MigrateSecurityGroupStrategy {
+
+  private AmazonClientProvider amazonClientProvider;
+
+  public AmazonClientProvider getAmazonClientProvider() {
+    return amazonClientProvider;
+  }
+
+  public DefaultMigrateSecurityGroupStrategy(AmazonClientProvider amazonClientProvider) {
+    this.amazonClientProvider = amazonClientProvider;
+  }
+
+
+  // TODO: The rest of this class does not belong here - move it into -nflx
+  @Override
+  public Set<MigrateSecurityGroupReference> getTargetReferences(SecurityGroupUpdater source,
+                                                                SecurityGroupLookup sourceLookup) {
+    if (source.getSecurityGroup().getGroupName().startsWith("nf-")) {
```

Comment:
I'd be ok with just making a configuration option for `ignoredSecurityGroupPrefixes` instead of pushing this out into a netflix-specific implementation but could go either way